src/middleware/isLoggedIn.js

```diff
@@ -1,17 +1,15 @@
-// const passport = require('passport');
-// const { Forbidden } = require('../utils/errors');
+const passport = require('passport');
```

Comment:
yes we do. from my understanding, it is through the login that we identify the user right?

Reply:
okay will accept.
asked because:
1. we could have required client to send us the author id and saved that instead but this works too
2. login implementation is not really part of the issue but I guess we have commented it before so we need to uncomment now.